datagrid/deploy.sh

```diff
@@ -23,7 +33,6 @@ while [ "$(oc get statefulset datagrid-service -o jsonpath='{.status.readyReplic
     echo "Waiting for statefulset to have $INSTANCES readyReplicas"
     sleep 5
 done
-oc exec datagrid-service-0 -- curl -X POST -d @/opt/datagrid/standalone/configuration/game-config.json -H application/json http://datagrid-service.datagrid-demo.svc.cluster.local:8080/rest/game/game
 
-# Set a native memory baseline
-./diagnostics/native-baseline.sh
+GAME_CONFIG_JSON=$(cat "$RESOURCE_DIR/config/game-config.json" | tr -d "\n")
```

Comment:
Wanted to change the game config the the most recent one without updating the image